Directory.Build.props

```diff
@@ -1,5 +1,11 @@
 <Project>
 
+  <PropertyGroup>
+    <!-- Non-SDK-style projects may import this file and then import Microsoft.Common.props.
```

Comment:
Is this only done by projects under src/.nuget? (Will we be able to remove this once those are SDK-style projects)?

Reply:
Yeah, that should be this case.